### DISCUSSION TITLE
✅ 🐞 By default, the button widget has a link

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#21190] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team.

As you know, even if we don't add a link in the button widget, the a tag is added in the front end codes. So, if we add the button widget inside a container that has a link, everything will break. I think this default behavior in button widget is not correct.

### Describe the solution you'd like

When we add the button, the button widget should not have a link by default. Whenever we consider a link for it, the a tag is added in the Html structure.
If you see the button widget in the bricks builder, when we add the button widget it has no link. But when we add a link for it, the a tag is added in the html structure.

**Comment by jamieburchell** [comment]
One workaround I found just now was to add the `.elementor-button` class to a text heading. It would be useful if we could more easily add a button (or at least apply the styles of a button) to an element inside a clickable loop template.

**Comment by nicholaszein** [comment]
Hi @Michaelgimii,

I'm not sure why you reported this as a bug here. But anyway... This issue was resolved with the release of Elementor Core v3.21.0, which added the Optimized Control Loading experiment.

If you still think this is an issue, please follow the guidelines for opening a new [🐞 Bug Report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml), so we can help you better.

Regards
